src/index.js

```diff
@@ -272,6 +272,22 @@ const jsPrettier = (tree, {rules: {indent, eol}, jsBeautifyOptions}) => {
   return prettier(tree);
 };
 
+const addLang = (tree, {rules: {lang}}) => {
```

Comment:
I seem to have missed one important feature of the fact that it is necessary to `bypass` nodes recursively and for the method `sortAttr` it is also necessary

For example, it will not pass the tests:
```js
<div><p>a<></p></div>
```
**`return`**
```js
<div lang="en"><p>a</p></div>
```
**`but should return`**

```js
<div lang="en"><p lang="en">a</p></div>
```
which suggests that not all tags need to do this

Reply:
ahh, it nice catch, yeah it should not be `map`. let me check